frontend/package.json

```diff
@@ -6,11 +6,18 @@
   "scripts": {
     "start": "webpack --mode development --watch",
     "build": "webpack --mode production",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint src/",
+    "lint-check": "eslint --max-warnings 0",
+    "format": "prettier --write src/",
+    "format-check": "prettier --check"
```

Comment:
Can you include an update to the readme with any common linting/formatting commands we should be using?